src/Microsoft.Security.Utilities.Core/DataClassification/DAT101_003_Integer.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Collections.Generic;
+
+namespace Microsoft.Security.Utilities
+{
+    public class Integer : RegexPattern
+    {
+        public Integer()
+        {
+            Id = "DAT101/003";
+            Name = nameof(Integer);
+            Pattern = @"^-?\d+$";
```

Comment:
>-? [](http://example.com/codeflow?start=25&length=2)

ha. nice catch. 

I suppose a prefixed `+` is not required. If we did add it we would create base64 confusion!

Reply:
but what about decimal values? hm. what about locale-rendered formatted numbers?

12.0
1,234,500